hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueHandler.java

```diff
@@ -1160,6 +1205,15 @@ public void closeContainer(Container container)
                 .getContainerID() + " while in " + state + " state.", error);
       }
       container.close();
+
+      KeyValueContainerData containerData =
```

Comment:
Can we move this logic into contianer.close()?

Reply:
Moved inside contianer.close()